fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java

```diff
@@ -487,6 +488,24 @@ public PlanFragment visitPhysicalTopN(PhysicalTopN<? extends Plan> topN, PlanTra
         return mergeFragment;
     }
 
+    private List<OrderKey> addAliasForNonSlot(List<OrderKey> orderKeys) {
```

Comment:
i think, we do not need to re-generate orderKey list. just mod generateTupleDesc to generate slot for non-SlotReference key is enough

Reply:
`this.addExprIdSlotRefPair(alias.getExprId(), slotRef);`
we need an ExprId to link slotRef. 
there is no exprId in expressions like `k % 5`, that why I wrap it into an alias to get the exprId